smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/NonModularServiceGenerator.java

```diff
@@ -70,7 +70,10 @@ public void run() {
 
         // Generate the client and extend from the modular client.
         writer.writeShapeDocs(service);
-        writer.openBlock("export class $L extends $T {", "}", nonModularName, serviceSymbol, () -> {
+        String serviceSymbolName = serviceSymbol.getName();
+        writer.addImport(serviceSymbolName, serviceSymbolName,
+                "./" + CodegenUtils.SOURCE_FOLDER + serviceSymbol.getNamespace().substring(1));
```

Comment:
```suggestion
        writer.addUseImports(serviceSymbol);
```

If that doesn't work, the symbol itself needs updated.

Reply:
> writer.addUseImports(serviceSymbol);

This doesn't work.

I need to update Symbol while creating it in SymbolVisitor right?
https://github.com/awslabs/smithy-typescript/blob/5a242c73bcfeac9d637dce3203dff90d[…]va/software/amazon/smithy/typescript/codegen/SymbolVisitor.java

Reply:
right